src/bmpwrite.c

```diff
@@ -333,22 +333,22 @@ void generate_BMP_RGB_data (struct context * context, unsigned char * offset_poi
   if ((context -> source -> color_format & PLUM_COLOR_MASK) == PLUM_COLOR_32)
     data = context -> source -> data;
   else {
-    data = ctxmalloc(context, sizeof *data * context -> source -> width * context -> source -> height);
-    plum_convert_colors(data, context -> source -> data, (size_t) context -> source -> width * context -> source -> height,
-                        PLUM_COLOR_32, context -> source -> color_format);
+    size_t size = (size_t) context -> source -> width * context -> source -> height;
```

Comment:
Factoring out a common `width * height` expression, as is done almost everywhere else.